docs/src/pages/comparing-astro-vs-other-tools.md

```diff
@@ -0,0 +1,136 @@
+
+Hugo offers "zero client-side JavaScript" by not including any JavaScript on your site by default. By making JavaScript opt-in, Hugo sites can be written with little to no JavaScript. However, when you do need client-side JavaScript, it's up to you to set up your own build pipeline. This can be time consuming, and is only as optimized as what you can build yourself with Webpack, Snowpack, or Vite.
+
+By contrast, Astro has built-in support for Partial Hydration and Island Architecture to only load the bare minimum amount of JavaScript in the browser. In production, Astro automatically optimizes your included component JavaSript bundles for you. This gives you built-in support to include individual JavaScript components on the page, without any effort on your part.
+
+#### Comparing Hugo vs. Astro Performance
+
+Hugo has few opinions about what you build, so it's difficult to compare generally. Because Hugo includes zero JavaScript by default, it is possible for an Hugo website to be as fast as an Astro website. However, because you are responsible for building your own build pipeline for any JavaScript that you include on the page, most Hugo websites will not be as well-optimized as Astro websites.
+
+
+## Jekyll vs. Astro
+
+[Jekyll](https://jekyllrb.com/) is a popular static site generator, powered by Ruby.
+
+Jekyll supports an older [templating languages](https://jekyllrb.com/docs/liquid/) for HTML called Liquid. Astro lets you write HTML using modern UI component libraries (React, Preact, Vue, Svelte) or a custom component syntax that is similar to HTML + JSX. Jekyll does not support using modern UI component libraries for HTML templating.
+
+Jekyll offers "zero client-side JavaScript" by not including any JavaScript on your site by default. By making JavaScript opt-in, Jekyll sites can be written with little to no JavaScript. However, when you do need client-side JavaScript, it's up to you to set up your own build pipeline. This can be time consuming, and is only as optimized as what you can build yourself with Webpack, Snowpack, or Vite.
+
+By contrast, Astro has built-in support for Partial Hydration and Island Architecture to only load the bare minimum amount of JavaScript in the browser. In production, Astro automatically optimizes your included component JavaSript bundles for you. This gives you built-in support to include individual JavaScript components on the page, without any effort on your part.
+
+#### Comparing Jekyll vs. Astro Performance
+
+Jekyll has few opinions about what you build, so it's difficult to compare generally. Because Jekyll includes zero JavaScript by default, it is possible for an Jekyll website to be as fast as an Astro website. However, because you are responsible for building your own build pipeline for any JavaScript that you include on the page, most Jekyll websites will not be as well-optimized as Astro websites.
```

Comment:
I would say that "it is easier to write  a well optimized site in Astro compared to Jekyll / all the other SSG's" rather than "most Jekyll websites will not be as well-optimized" unless if we have data to back that up.

Reply:
yup, this was too strong to back up. rewrote